authentication-saml/src/main/java/com/synopsys/integration/alert/authentication/saml/web/SAMLGlobalConfigController.java

```diff
@@ -0,0 +1,48 @@
+package com.synopsys.integration.alert.authentication.saml.web;
+
+import com.synopsys.integration.alert.api.common.model.ValidationResponseModel;
+import com.synopsys.integration.alert.authentication.saml.model.SAMLGlobalConfigModel;
+import com.synopsys.integration.alert.common.rest.AlertRestConstants;
+import com.synopsys.integration.alert.common.rest.api.StaticUniqueConfigResourceController;
+import com.synopsys.integration.alert.common.rest.api.ValidateController;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Map;
+import java.util.UUID;
+
+@RestController
+@RequestMapping(AlertRestConstants.SAML_PATH)
+public class SAMLGlobalConfigController implements StaticUniqueConfigResourceController<SAMLGlobalConfigModel>, ValidateController<SAMLGlobalConfigModel> {
+    @Autowired
+    public SAMLGlobalConfigController() {
+        // TODO Wire actions
+    }
+
+    @Override
+    public SAMLGlobalConfigModel create(SAMLGlobalConfigModel resource) {
+        return new SAMLGlobalConfigModel(UUID.randomUUID().toString());
+    }
+
+    @Override
+    public SAMLGlobalConfigModel getOne() {
+        return new SAMLGlobalConfigModel(UUID.randomUUID().toString());
```

Comment:
Why not create a static member variable for the UUID

`private static final UUID DEFAULT_ID = UUID.randomUUID();`
Then just use the static in the create and get one methods since this API should always use the same UUID.  Once this uses data from the database then the static member variable can be removed.

Reply:
Updated